### DISCUSSION TITLE
Update mobile app to enforce SOC2 security protocols like screen timeout, etc

### DISCUSSION THREAD

**Discussion by maebeale** [community/community#16529] [Mobile]
My workplace won't approve of us using github on mobile (web or native app) bc they can't enforce all the security required (lock screen, timeout, lost device process, etc). Do you have anything on the roadmap to offer enhanced security so managers like me can browse and review code from my phone?

**Comment by brianlovin** [comment]
Thanks for the message @maebeale — I'll flag this to our team, but in order to answer correctly it might be useful to have the specific requirements for mobile apps at your organization. For example, we don't support biometrics for the app itself, but the device itself has biometrics to unlock the phone. We also have a way to remotely revoke sessions through a user's security settings (https://github.com/settings/security) but there might be more specific managed needs at the company level.

**Reply by byennen** [reply]
This is a big issue for us. Any update on this? SSO has made this stop working and it's a SOC2 requirement for us. Thanks for the help!

**Comment by AzureCloudMonk** [comment]
Why does the GitHub Mobile iOS app not have an “App Lock” requiring a screen lock when you open the app using the Touch ID, mobile facial recognition or at least a mobile device PIN. Microsoft Authenticator, LastPass, and many other two-factor methods such as Authenticator apps require this. Since GitHub Mobile app is now an “Authenticator app” it should also have this security feature.

**Comment by aconsuegra** [comment]
This feature is now available on GitHub Mobile! Check the announcement in https://github.blog/changelog/2024-08-06-app-lock-via-face-id-or-biometrics/, and let us know if you have any feedback. Thanks for your patience!

**Comment by Abdulrehmanatif** [comment]
GitHub does not currently have specific plans on its roadmap for enhanced security features tailored to mobile usage that would address concerns like lock screens, timeout settings, or lost device processes. However, GitHub often updates its security features and policies, so it’s a good idea to keep an eye on their [official blog](https://github.blog) or their [security page](https://github.com/security).

For now, you might consider implementing additional security measures on your own devices, such as enabling strong password protection, using two-factor authentication, and ensuring that your device itself is secured with a lock screen and remote wipe capabilities. If mobile access is crucial, you could also discuss with your IT department about developing a specific policy or solution that meets both your workplace's security requirements and your need to access GitHub from your phone.
